app/models/user.rb

```diff
@@ -45,7 +45,7 @@ def self.change_code(type, newCode)
   end
 
   ##Methods dealing with search
-  @@member = ["first", "last", "team", "major", "skillset", "linkedinLstring", "facebook", "year"]
```

Comment:
Fair point. Users would simply search for a user's name to find their professional social media.